### ISSUE TITLE
Performence doesn't change

### ISSUE THREAD

**Issue by albert0713** [GuYuc/WS-DAN.PyTorch#25]
the val acc doesn't change for several epoch when I train the model. What's wrong with it?
Epoch 85/300: 100%|██████████| 8144/8144 [1:24:14<00:00,  1.61 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.7789, Val Acc (0.52, 2.70)]
Epoch 86/300: 100%|██████████| 8144/8144 [1:24:12<00:00,  1.61 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.8678, Val Acc (0.52, 2.70)]
Epoch 87/300: 100%|██████████| 8144/8144 [1:24:20<00:00,  1.61 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.6306, Val Acc (0.52, 2.72)]
Epoch 88/300: 100%|██████████| 8144/8144 [1:24:12<00:00,  1.61 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.8817, Val Acc (0.52, 2.71)]
Epoch 89/300: 100%|██████████| 8144/8144 [1:23:19<00:00,  1.63 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.9679, Val Acc (0.52, 2.71)]
Epoch 90/300: 100%|██████████| 8144/8144 [1:22:45<00:00,  1.64 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.4493, Val Acc (0.52, 2.71)]
Epoch 91/300: 100%|██████████| 8144/8144 [1:22:45<00:00,  1.64 batches/s, Loss 6.2739, Raw Acc (0.39, 2.49), Crop Acc (0.39, 2.49), Drop Acc (0.39, 2.49), Val Loss 44.8331, Val Acc (0.52, 2.67)]
Epoch 92/300:  97%|█████████▋| 7933/8144 [1:09:51<01:57,  1.80 batches/s, Loss 6.2741, Raw Acc (0.39, 2.53), Crop Acc (0.39, 2.53), Drop Acc (0.39, 2.53)]

**Comment by zhwzhao** [comment]
I meet the same problem, have you solved it?  thanks for giving the answer.

```
Epoch 1/160:   0%|          | 0/2036 [00:00<?, ? batches/s]Start ModelCheckpoint
Epoch 1/160: 100%|██████████| 2036/2036 [16:00<00:00,  2.12 batches/s, Loss 11.7847, Raw Acc (0.56, 2.76), Crop Acc (0.47, 2.75), Drop Acc (0.49, 2.75), Val Loss 10.6813, Val Acc (0.61, 2.82)]
Epoch 2/160: 100%|██████████| 2036/2036 [15:32<00:00,  2.18 batches/s, Loss 9.6784, Raw Acc (0.53, 2.52), Crop Acc (0.43, 2.49), Drop Acc (0.43, 2.41), Val Loss 6.4371, Val Acc (0.46, 2.40)]
Epoch 3/160: 100%|██████████| 2036/2036 [15:32<00:00,  2.18 batches/s, Loss 11.2258, Raw Acc (0.44, 2.48), Crop Acc (0.44, 2.47), Drop Acc (0.43, 2.48), Val Loss 13.8269, Val Acc (0.56, 2.84)]
Epoch 4/160: 100%|██████████| 2036/2036 [15:32<00:00,  2.18 batches/s, Loss 14.1061, Raw Acc (0.52, 2.65), Crop Acc (0.55, 2.57), Drop Acc (0.52, 2.59), Val Loss 14.0123, Val Acc (0.45, 2.61)]
Epoch 5/160: 100%|██████████| 2036/2036 [15:52<00:00,  2.14 batches/s, Loss 12.9352, Raw Acc (0.53, 2.53), Crop Acc (0.52, 2.50), Drop Acc (0.50, 2.59), Val Loss 12.8567, Val Acc (0.46, 2.55)]
Epoch 6/160: 100%|██████████| 2036/2036 [16:08<00:00,  2.10 batches/s, Loss 12.8473, Raw Acc (0.65, 3.07), Crop Acc (0.63, 3.13), Drop Acc (0.60, 3.05), Val Loss 12.6061, Val Acc (0.61, 2.91)]
Epoch 7/160: 100%|██████████| 2036/2036 [16:02<00:00,  2.12 batches/s, Loss 11.9080, Raw Acc (0.72, 3.14), Crop Acc (0.69, 3.08), Drop Acc (0.69, 2.87), Val Loss 10.8328, Val Acc (0.85, 3.07)]
Epoch 8/160: 100%|██████████| 2036/2036 [15:35<00:00,  2.18 batches/s, Loss 11.9076, Raw Acc (0.64, 3.16), Crop Acc (0.61, 3.11), Drop Acc (0.61, 2.97), Val Loss 12.9705, Val Acc (0.99, 3.79)]
Epoch 9/160: 100%|██████████| 2036/2036 [15:40<00:00,  2.17 batches/s, Loss 11.0488, Raw Acc (0.87, 3.75), Crop Acc (0.86, 3.52), Drop Acc (0.66, 3.57), Val Loss 10.6276, Val Acc (1.07, 3.71)]
```

**Comment by GuYuc** [comment]
@albert0713 Hi. It seems that the training loss does not change after all these iterations. Manually dropping your learning rate might help, but i'm not sure. Such situations didn't occur in my machine.
